apps/core-app/src/components/GovernanceSettings.tsx

```diff
@@ -0,0 +1,172 @@
+import styled from 'styled-components';
+import { H3, H4, DataIndicator, ParSm, widthQuery, Theme } from '@daohaus/ui';
+
+import { TDao } from '../contexts/DaoContext';
+import {
+  formatPeriods,
+  formatValueTo,
+  fromWei,
+  getNetwork,
+  INFO_COPY,
+} from '@daohaus/common-utilities';
+import { useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+
+const GovernanceContainer = styled.div`
+  .tokens {
+    margin-top: 3rem;
+  }
+  h4 {
+    margin-top: 4rem;
+  }
+`;
+
+// putting this in place for when we bring in the action button
+const GovernanceCardHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  margin-bottom: 3rem;
+`;
+
+const DataGrid = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  align-content: space-between;
+  div {
+    margin-top: 3rem;
+    width: 34rem;
+
+    @media ${widthQuery.sm} {
+      min-width: 100%;
+    }
+  }
+`;
+
+const TokenDataGrid = styled(DataGrid)`
+  div {
+    width: 22.7rem;
+  }
+`;
+
+const StyledLink = styled.a`
+  text-decoration: none;
+  color: ${({ theme }: { theme: Theme }) => theme.link.color};
+`;
+
+type GovernanceSettingsProps = {
+  dao: TDao;
+};
+
+export const GovernanceSettings = ({ dao }: GovernanceSettingsProps) => {
+  const { daochain } = useParams();
+  const networkData = useMemo(() => {
+    if (!daochain) return null;
+    return getNetwork(daochain);
+  }, [daochain]);
+
+  return (
+    <GovernanceContainer>
+      <GovernanceCardHeader>
+        <H3>Governance</H3>
```

Comment:
this suppose to be `Governance Settings`?
![Screen Shot 2022-08-05 at 2 35 53 PM](https://user-images.githubusercontent.com/5998100/183158823-9a21e292-6b7b-4f9f-bd1f-c29173e53075.png)

Reply:
Yep - updating